pelican/__init__.py

```diff
@@ -384,7 +389,9 @@ def get_instance(args):
 
 def main():
     args = parse_arguments()
-    init(args.verbosity, args.fatal)
+    logs_deduping_min_level = getattr(logging, args.logs_deduping_min_level)
+    init_logging(args.verbosity, args.fatal,
+                 logs_deduping_min_level=logs_deduping_min_level)
```

Comment:
Change these references to: `log-dedup-min-level`

Reply:
Done